src/channels.rs

```diff
@@ -60,9 +41,7 @@ impl Runtime for TaskPoolRuntime {
     type Sleep = Pin<Box<dyn Future<Output = ()> + Send>>;
 
     fn spawn<F: Future<Output = ()> + Send + 'static>(&self, f: F) {
-        let task = self.pool.spawn(Box::pin(f));
-        #[cfg(not(target_arch = "wasm32"))]
-        self.tasks.lock().unwrap().push(task);
+        self.pool.spawn(Box::pin(f)).detach();
```

Comment:
Are you sure this will not cause a dangling thread if someone manually destroys Bevy App.

Reply:
That's interesting. I have not thought about that. I can investigate and maybe find a better suited pattern that can please that condition and clippy.

Reply:
Yeah. Before states were a thing, people used to shut down a "menu" bevy app and proceed to "gameplay" bevy app, configured differently.

Reply:
Hi, this is about this [piece of code](https://github.com/smokku/bevy_networking_turbulence/blob/3deab3ccad69a5d81294c6bdc605f578431ccc37/src/transport.rs#L340). I don't understand why you spawn a task and immediately drop it if the code is compiled to wasm.

Reply:
On WASM tasks are spawned as `async` and are tracked by JS runtime. On native these are real threads and we need to keep a live object, because `drop` shuts down a thread.